CHANGELOG.md

```diff
@@ -17,6 +17,14 @@ This file is used to list changes made in each version of the AWS ParallelCluste
 - Do not wait for static nodes in maintenance to signal CFN that the head node initialization is complete.
 - Upgrade `aws-cfn-bootstrap` to version 2.0-28.
 - Upgrade Python to 3.9.17.
+- Use OpenRM as NVIDIA kernel module for Linux instead of NVIDIA closed source module.
```

Comment:
we should move Nvidia change on top of the list,or at least before aws-cfn-bootstrap and python upgrades.